### PR TITLE
Update UG, DG and my PPP

### PR DESCRIPTION
**Pull request by bakano98**
Here's a list of things done

For the UG:
1. Changed all instances of `Index` to `INDEX` for better consistency across the UG
2. Changed all instances of `ModuleMate Finder` to `ModuleMateFinder` for better consistency across the UG. Do check your PPPs and see if it's using `ModuleMateFinder`. Let's standardise to this.
3. Added a list of commands that `undo` and `redo` applies to in the UG. Do add on if there's anything missing from there.
4. Updated UI image to look better

For the DG:
1. Update `Model`'s link to point to our GitHub repo
2. Update user stories to include things that we have implemented
3. Update and reorder use cases. Reordering is done to follow the list of commands in the UG. Additional use cases were added for `archive`, `unarchive`, `switch`, `clearmodules`, `deletemodules`. Not too sure if I missed anything out, but similarly if I missed out anything, please feel free to add on.

I also updated my PPP. Reminder to update your PPP if you haven't already!

### CODE REVIEW

**Comment by bakano98** [comment]
Additionally, I have checked through the UG/DG for anything that might be missing but I think it's all good for now!

File: docs/UserGuide.md [hunk 1]
```diff
@@ -485,6 +487,25 @@ Examples:
...
+* `clear`
+* `clearmodules`
+* `delete`
+* `deletemodules`
+* `sort`
+* `switch`
+* `archive`
+* `unarchive`
+* `undo`/`redo` i.e. you can `undo` a `redo`, and you can `redo` an `undo`
+</div>
```
Comment (lawwm) [inline docs/UserGuide.md:507]: Undo has a list of commands as well that is in different order to the one in Redo. Maybe just keep one list for both, or make the list in undo identical to the list in redo?

**Review by lawwm — COMMENTED** [review]
LGTM

File: docs/UserGuide.md (hunk 1 above)
Comment (bakano98) [reply docs/UserGuide.md:507]: Good point. I'll remove the one in Redo and keep this list since it's consistent with the formatting in shortcuts.

**Review by Btaykb — COMMENTED** [review]
LGTM

**Pull request merged**